### DISCUSSION TITLE
blackarch error

### DISCUSSION THREAD

**Discussion by khulay** [BlackArch/blackarch#3226] [Q&A]
i already installed the newest version of blackarch, and when i'm rebooting my laptop doesn't booting to blackarch, and just stuck in boot mode, how i can fix that? and how i can got iso version 2020.12.1? thanks for the answer.... ^_^

**Comment by ikstream** [comment]
Please open an issue for a bug report. Fill it please as extensive as possible so we can try to recreate your issue. 

Also maybe provide an image where the boot process stops.

- Please also provide infos about the iso image you used (slim/net/full) .

- Have there been any error messages during install or anything unusual?


The old iso isn't available anymore. 

**On a site note:** BlackArch is rolling release, so if you had a working install, you don't need to install the whole system on a new iso release. Just upgrade regularly with `pacman -Syu` at the minimum once a month to avoid issues.

**Reply by khulay** [reply]
i used BlackArch Full iso image, there is no messages error during install, blackarch already successfull to install but when i'm rebooting my laptop doesn't booting to blackarch, i already install that's full iso image 3 times using Ventoy, Rufus, and Balenaetcher, but in the end still have the same problem, cant boot in to the blackarch again, what should i do?

**Comment by ikstream** [comment]
Where in the boot process does your system stop?
Does it show the Systemd version in use and the clean blocks and blanks?
Does that describe your issue? If not, please describe where during the boot process your system gets stuck.

**Reply by khulay** [reply]
my laptop system does stop in black screen session, and that's not shown any systemd version, it's just blank and black screen on showing in my laptop

**Comment by ikstream** [comment]
In addition, when you were using Rufus, did you use DD mode?

**Reply by khulay** [reply]
yeah, i used DD mode

**Comment by ikstream** [comment]
Which install mode did you select (online/offline)?
Did you enable encryption?
Have you tried the slim iso as well? Does the same error occur?

**Reply by khulay** [reply]
i choose offline install mode because i used full iso version, yeah i enable encryption, yeah i tried the slim as well and that's no errors at all
in addition, can i add all the tools in the full version to the slim version? thank you :)

**Reply by noraj** [reply]
>  in addition, can i add all the tools in the full version to the slim version? thank you :)

Yes, but do you really need 3000 tools? You will probably never used them all.

Either use the netinstall ISO to install only what you need, or use the slim ISO to have a base and add what you need.

**Reply by ikstream** [reply]
> [...] yeah i tried the slim as well and that's no errors at all

So i would stick with it, and just install the tools you want to use. The slim was designed to be easy to use and ease the install process.

> [...] can i add all the tools in the full version to the slim version? 

Yes, there is really no difference between full, slim and net iso when it comes to tool availability. They all use our repository. The only difference is the number of tools and environments they come with preinstalled.
